### DISCUSSION TITLE
Need help with loco positioning system.

### DISCUSSION THREAD

**Discussion by Fygas1** [bitcraze/discussions#1445] [Q&A] [answered]
Hey, I'm trying to set up the loco positioning system and I'm having some trouble. Here is my setup: https://imgur.com/a/oWsvzFP

I'm using 8 anchors, 4 at the bottom and 4 at the top. The anchors are set. The x- axis is almost 4.5 meters long and the y-axis almost 5.5 meters long. The distance between the top ones and the bottom ones are 2.5 meters long.

When I place my Crazyflie in the middle of the area the Crazyflie client shows the position pretty accurately. The thing is when placing the Crazyflie in the corner of the LPS area, the position readings are inaccurate and fluctuate significantly, showing it as being near the corner one moment and then far away or even out of bounds the next. All nodes are updated so the latest firmware. Here's a video I recording showcasing what I just said: https://streamable.com/zgxruf

Is there anything wrong with my setup? Do I need to move any objects? The bottom anchors are placed on glasses and the top ones are hanging from the roof.

Also another problem I have is when I try to run the autonomousSequence.py code (https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/autonomy/autonomousSequence.py).
So I place my Crazyflie in the center of the LPS area. Then I enter only 1 position in my code inside the sequence array. I enter the x and y of the current position of the Crazyflie (so somewhere in the middle) and I enter a higher Z so that the Crazyflie can take off and hover at the center of the LPS area. I run the code, the Crazyflie lifts off correctly, but then starts moving towards the wall. Why does this happen if I only entered 1 position inside the sequence array and that position basically tells it to hover and not move along the x or y axis?

Thanks in advance

**Comment by Fygas1** [comment]
Forgot to mention that I do not have the flow deck installed. Just the Loco Deck. Do I also need the flow deck?

**Comment by Fygas1** [comment, marked as answer]
Nevermind fixed it.

**Reply by tobbeanton** [reply]
I guess that means everything is fixed. Reopen it if it is not.

**Reply by danielcmdeveloper** [reply]
Good afternoon,
I am interested to know the solution to this problem, as we have something similar.
Can anyone tell me the solution that was adopted?
Thank you very much.
Best regards

**Reply by Fygas1** [reply]
Hey,

It's been some time now so I cant recall the exact details but I believe I had the wrong anchors numbering in the client. For example I had my anchor 4 in my setup as 3 inside the client. So the anchor X,Y measurements were wrong inside the client. After fixing that I think it worked.

**Reply by danielcmdeveloper** [reply]
Thank you very much for the reply.
I will check the numbering again in case the problem is the same in this case.
Regards
:-)

**Reply by knmcguire** [reply]
@danielcmdeveloper it would be more helpful to start a new discussion thread, eventhough if your problem seems to be similar
